docs/faq-generated-code.md

```diff
@@ -0,0 +1,37 @@
+# Generated Code FAQ
```

Comment:
@programm-ingovals @dethell I'd appreciate your feedback on this FAQ.

Reply:
This is a good improvement. I've had several developers wonder what they are supposed to edit and this clarifies that well.